code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -315,3 +315,26 @@
 
 /mob/living/simple_animal/mouse/plague
 	disease_carrier = 1
+
+/mob/living/simple_animal/mouse/mouse_op
+	name = "mouse operative"
+	namenumbers = FALSE
+	min_oxy = 0
+	minbodytemp = 0
+	maxbodytemp = 5000
+	maxHealth = 50
+	health = 50
+	universal_speak = 1
+	can_chew_wires = 1
+	mutations = list(M_NO_SHOCK)
+
+/mob/living/simple_animal/mouse/mouse_op/New()
+	..()
+	desc = "Oh no..."
```

Comment:
I guess I'm repeating Damian here but why is the `desc` in `New()` when it's in the type definition for the base mouse mob? Is this all done so that if you run `initial()` on the mouse op it'll become a regular mouse? I don't get it.

Reply:
I call the parent type's New() because it sets fallback icons that shouldn't matter but I wasn't sure if they're checked anywhere else so it's _just in case_. The parent New() overwrites the icon_state, desc, and universal_understand vars so I have to re-set them in the child's New().

The alternative solution would be to completely override the child's New(), but that would require copypasting lines that already do what I want.

Reply:
Ah, I see how it is. Shitcode like this (the original mouse code) makes me wish there was some way to call the grandparent's `New()`, skipping the parent entirely...

Reply:
I was thinking about refactoring it a bit but mouse rework will have to wait since there's more important projects I want to do first.